cumulusci/core/config/BaseProjectConfig.py

```diff
@@ -546,16 +546,37 @@ def process_github_dependency(self, dependency, indent=None, include_beta=None):
             repo_name = repo_name[:-4]
         repo = gh.repository(repo_owner, repo_name)
 
-        # Determine the ref if specified
-        kwargs = {}
-        if "tag" in dependency:
-            tag = dependency["tag"]
-            kwargs["ref"] = tag
+        # Determine the commit
+        release = None
+        if "ref" in dependency:
+            ref = dependency["ref"]
         else:
-            tag = None
+            if "tag" in dependency:
+                try:
+                    # Find the github release corresponding to this tag.
+                    release = repo.release_from_tag(dependency["tag"])
+                except NotFoundError:
+                    raise DependencyResolutionError(
+                        "{}No release found for tag {}".format(
+                            indent, dependency["tag"]
+                        )
+                    )
+            else:
+                release = self._find_latest_release(repo, include_beta)
+            if release:
+                ref = repo.tag(
+                    repo.ref("tags/" + release.tag_name).object.sha
+                ).object.sha
```

Comment:
`release.target_commitish` ?

Reply:
I hoped so too, but that returns "master"